src/assets/stylesheets/base/_icons.scss

```diff
@@ -35,7 +35,6 @@
   line-height: 1;
   text-transform: none;
   white-space: nowrap;
-  speak: none;
```

Comment:
to be clear, I removed this because it's not actually a valid CSS rule. I did not replace it with a value of the (new?) `speak-as` because there is no equivalent to `none` in there. Ref: https://www.w3.org/TR/css-counter-styles-3/#descdef-counter-style-speak-as

Reply:
You're right, its not valid and we should remove it. I included it originally because it was part of the original Material Design icon classes. `speak-as` is a descriptor, not a property, and thus only valid within `@counter-style` at rules.